stdpopsim/models.py

```diff
@@ -124,6 +124,7 @@ def __init__(self):
         self.demographic_events = []
         # Defaults to a single population
         self.migration_matrix = [[0]]
+        self._generation_time = -1
```

Comment:
I don't see the point of making this a 'private' property - everything else is represented as a standard attribute so I think we should do the same for ``generation_time``. Why make a special case of generation_time?

Reply:
yeah no reason I guess. I added it as a class `property` of each model emulating `_name`. can switch it.